charts/ups-broker/templates/broker-service.yaml

```diff
@@ -0,0 +1,16 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
```

Comment:
The service name will end up being `<release name>-<chart name>` == `<release name>-ups-broker`.

This PR modifies the example broker resource used in the walkthrough so the URL of the broker it queries for the catalog is `http://ups-broker-ups-broker.ups-broker.svc.cluster.local`.

The practical implication of this is that when we refactor the walkthrough, it should specify, precisely, the following command for installing the ups-broker chart:

```
helm install charts/ups-broker --name ups-broker --namespace ups-broker
```